pkg/util/gzip64/gzip64.go

```diff
@@ -0,0 +1,51 @@
+package gzip64
+
+// Package for manipulating base64-encoded gzip'd data.
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"strings"
+)
+
+// Encode returns the base64 string representing the gzip compressed data
+func Encode(data []byte) string {
+	buf := &bytes.Buffer{}
+
+	b64w := base64.NewEncoder(base64.StdEncoding, buf)
+	gzw := gzip.NewWriter(b64w)
```

Comment:
Maybe use https://pkg.go.dev/compress/gzip#NewWriterLevel with https://pkg.go.dev/compress/gzip#BestCompression to buy a bit more time...

Reply:
Unclear to me how much benefit this provides -- it's a good suggestion though, and we can look into it if we see a need. Thanks!